sonar-kotlin-plugin/build.gradle.kts

```diff
@@ -99,7 +99,7 @@ tasks.shadowJar {
     exclude("org/jetbrains/kotlin/org/jline/**")
     exclude("org/jetbrains/kotlin/net/jpountz/**")
     doLast {
-        enforceJarSizeAndCheckContent(shadowJar.get().archiveFile.get().asFile, 33_400_000L, 37_000_000L)
+        enforceJarSizeAndCheckContent(shadowJar.get().archiveFile.get().asFile, 36_900_000L, 37_500_000L)
```

Comment:
Just curious, was it less than 37M after you removed the `companion object`?

Reply:
Yes, only about 100 bytes or so